.env.example

```
@@ -65,29 +66,48 @@ USE_REAL_TRANSLATION=false
+TTS_LOCKOUT_BUFFER_MS=200
+
+NLLB_MODEL_PATH=AIRFLOW-MODEL-PATH
+QWEN_VL_MODEL_PATH=AIRFLOW-MODEL-PATH
+WHISPER_MODEL_PATH=AIRFLOW-MODEL-PATH
+PIPER_TTS_ES_PATH=AIRFLOW-MODEL-PATH
+PIPER_TTS_PT_PATH=AIRFLOW-MODEL-PATH
+PIPER_TTS_EN_PATH=AIRFLOW-MODEL-PATH
+SILERO_VAD_MODEL_PATH=AIRFLOW-MODEL-PATH
 
 # ── Auth (GCIP / Firebase) ────────────────────────────────────────────────────
 GCIP_API_KEY=                               # Firebase Console → Project Settings → General → Your apps
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)